.github/workflows/orders_pipeline.yaml

```diff
@@ -0,0 +1,47 @@
+name: orders_pipeline
+
+on:
+  push:
+    paths:
+      - 'src/orders/**'
+    branches:
+      - 'main'
+      - 'feature**'
+  workflow_dispatch:
+jobs:
+  run_unit_tests:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11' 
+      - name: Install dependencies
+        run: |
+          make install
+        working-directory: ./tests
+      - name: Run tests
+        run: |
+          make test
+        working-directory: ./tests
+
+  development:
+    if: startsWith(github.event.ref, 'refs/heads/main')
+    uses: ./.github/workflows/pipeline_template.yaml
+    needs: [run_unit_tests]
+    with: 
+      stack_name: sam-hiberry-app-orders-development
+      sam_deploy_overrides: "ShopifyEventBusName=aws.partner/shopify.com/71471267841/InfinityCloudSource StageName=development"
+      sam_template: src/orders/template.yaml
+      aws_region: us-east-1
+      pipeline_execution_role: arn:aws:iam::937030943129:role/aws-sam-cli-managed-dev-pipel-PipelineExecutionRole-43mYyIsd3U1C
```

Comment:
is there a way of using variables or even secrests?

Reply:
Changes applied to use variables